### ISSUE TITLE
"No permission to access this resource" with account A should have a button for easy switching between accounts

### ISSUE THREAD

**Issue by megoth** [solid/node-solid-server#877]
As people might have multiple WebIDs, it should be easy to switch account if you're trying to access a resource that is not shared with your account A, but is with account B.

(If I'm misunderstanding how WebID handles this case, please let me know)

**Comment by rubensworks** [comment]
We have a logout button now on those pages. So I assume this issue can be closed then?

**Comment by kjetilk** [comment]
Yeah, this has been in the OpenLink workflow, so I suppose it is good?

**Comment by timbl** [comment]
@rubensworks When you say "We have a logout button now on those pages" - on which version of what?  It certainly is not currently on solid.community at the moment.  Or on timbl.com.

**Comment by TallTed** [comment]
The way this works in the OpenLink workflow is via our browser extensions ([YouID](http://youid.openlinksw.com/) and/or the [OpenLink Structured Data Sniffer (OSDS)](http://osds.openlinksw.com/)) and [WebID+TLS+Delegation](http://vos.openlinksw.com/dataspace/owiki/wiki/VOS/WebIDTLSDelegationWhatWhyHow).

So far as I am aware, there is nothing in the server that does (nor can do) anything about this, and this cannot be done with an un-extended browser, as none of them have this functionality built in -- because all browsers have only one TLS login per server, per browser instantiation.

**Comment by rubensworks** [comment]
@timbl It's been a while, but this logout button was added in the 5.0.0 release if I remember correctly.
I thought it was required to manually update some of the (template) views, which may not have happened on solid.community and timbl.com.
